src/OrchardCore.Modules/OrchardCore.DynamicCache/Manifest.cs

```diff
@@ -5,6 +5,6 @@
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Dynamic Cache.",
+    Description = "The Dynamic Cache module allows to use tag helpers to cache items dynamically.",
```

Comment:
Or perhaps: `"The Dynamic Cache module adds tag helpers to cache items dynamically."`?

Reply:
ok